src/types/ProposalVersion.ts

```diff
@@ -5,22 +5,24 @@ import type {
 	ProposalFieldValue,
 	WritableProposalFieldValuedWithProposalVersionContext,
 } from './ProposalFieldValue';
+import type { Writable } from './Writable';
 
-export interface ProposalVersion {
-	id: number;
+interface ProposalVersion {
+	readonly id: number;
 	proposalId: number;
+	readonly version: number;
 	applicationFormId: number;
-	version: number;
-	fieldValues?: ProposalFieldValue[];
-	createdAt: string;
+	readonly fieldValues: ProposalFieldValue[];
+	readonly createdAt: string;
 }
 
-export type ProposalVersionWrite = Omit<
-	ProposalVersion,
-	'createdAt' | 'fieldValues' | 'id' | 'version'
-> & { fieldValues: WritableProposalFieldValuedWithProposalVersionContext[] };
+type WritableProposalVersion = Writable<ProposalVersion>;
+
+type WritableProposalVersionWithFieldValues = WritableProposalVersion & {
+	fieldValues: WritableProposalFieldValuedWithProposalVersionContext[];
```

Comment:
It looks like a typo here carried over from previous commits on the other PR's branch.